src/Storage/SharedLock.cs

```diff
@@ -43,5 +44,11 @@ public static string Unlock(int contentId, string @lock)
         {
             return Storage.DeleteSharedLock(contentId, @lock);
         }
+
+        public static void Cleanup()
+        {
+            SnTrace.Write("Cleanup shared locks.");
```

Comment:
This line will write to the `Custom` trace category. We should use one of the built-in categories.

Reply:
Fixed.